### ISSUE TITLE
Cannot set personal SK for ldap users

### ISSUE THREAD

**Issue by Libeccio84** [nilsteampassnet/TeamPass#1712]
Dear all,
I have recently tried the master version of TeamPass with ldap integration.
I can successfully log in with my LDAP account, but when I try to set my SK and I confirm it, I get this error:
![screen shot 2017-03-29 at 12 34 33](https://cloud.githubusercontent.com/assets/23318355/24450852/c2b4c422-147c-11e7-9a8b-fc9348953e93.png)

On the other hand, If I do the same with a local user created with the admin account, this error does not occur. Any idea?

My server is a CentOS7 with MariaDB 5.5.52 and PHP 7.0.16

**Comment by nilsteampassnet** [comment]
Can you please share the apache log error?

**Comment by roru69** [comment]
I have a similair problem. I authenticate via Active Directory. After that, when I create an entry in my personal store, it isn't possible because the personal salt key isn't set.

When i set the key (Reset Personal Salt Key) it seems the key is set, but when i try to make an entry in my personal password store it says that i need to set my salt key (Again)

[error.txt](https://github.com/nilsteampassnet/TeamPass/files/967739/error.txt)

Kind regards,

Roru69

.

When i use the option (Change my personal saltkey) and enter the old and the new one it results in an "Internal Server Error 500" See attached php error log.

**Comment by roru69** [comment]
Tested a bit more and when I disable LDAP and 2 Factor Authentication the problem persists. I created another testuser and I'm still not able to create a new item in my personal store. Same errors in the logging. So in my opinion it is not related to LDAP. 

[error.txt](https://github.com/nilsteampassnet/TeamPass/files/967837/error.txt)

Kind regards,

Roru69

Debian 8.7 - Apache 2.4.10 - MySql Ver 14.14 Distrib 5.5.55 - PHP 5.6.30-0 - Teampass 2.1.27
